dd-sdk-android/src/test/kotlin/com/datadog/android/core/internal/data/upload/DataUploadRunnableTest.kt

```diff
@@ -86,8 +86,8 @@ internal class DataUploadRunnableTest {
             )
         whenever(mockNetworkInfoProvider.getLatestNetworkInfo()) doReturn fakeNetworkInfo
         val fakeSystemInfo = SystemInfo(
-            batteryStatus = forge.aValueFrom(SystemInfo.BatteryStatus::class.java),
-            batteryLevel = forge.anInt(20, 100)
+            batteryFullOrCharging = true,
+            batteryLevel = forge.anInt(min = 90, max = 100)
```

Comment:
why are you changing the min value here ?

Reply:
not sure why, probably was a left over from pervious iterations, will put it back